assets/js/theme/global/modal.js

```diff
@@ -230,6 +234,10 @@ export class Modal {
     }
 
     onModalOpened() {
+        if (this.pending) {
+            this.$modal.one(ModalEvents.loaded, () => this.setupFocusTrap());
+        } else this.setupFocusTrap();
```

Comment:
I would use/remove  curly brackets for both parts of `If statement` to stay consistent

Reply:
It was done to make code 2 strings less. According to you `else` should never be used without curly brackets, because it can not be used without `if` :)

Reply:
it's not according to me)) Our ESLint extends airbnb style guides and we have no separate rule for blocks. According to [guide](https://github.com/airbnb/javascript#blocks--braces) it's a good practice to "Use braces with all multiline blocks".
ESLint also has its [rule](https://eslint.org/docs/rules/nonblock-statement-body-position) about block (you can check `Examples of correct code for this rule with the default "beside" option`)

Reply:
Saw such styling on freecodecamp but as we work with an eye on airbnb guide - agree with you. updated &#9989;